std/socket.d

```diff
@@ -2160,8 +2160,8 @@ struct TimeVal
 /**
  * A collection of sockets for use with $(D Socket.select).
  *
- * $(D SocketSet) wraps the platform $(D fd_set) type. However, unlike
  * $(D fd_set), $(D SocketSet) is not statically limited to $(D FD_SETSIZE)
+ * $(D SocketSet) wraps the platform $(D fd_set) type. However, unlike
```

Comment:
Argh. My text editor has an (overly) convenient key combo for "swap lines". Must've hit it my mistake without noticing. Will fix.